asn1ate/pyasn1gen.py

```diff
@@ -474,8 +482,8 @@ def generate_OID(self):
         return str(fragment)
 
 
-def generate_pyasn1(sema_module, out_stream):
-    return Pyasn1Backend(sema_module, out_stream).generate_code()
+def generate_pyasn1(sema_module, out_stream, referenced_modules = None):
```

Comment:
Please get rid of the default value for referenced_modules here, we're only calling this from one place, and we have them available.